applications/portal/templates/deployment.yaml

```diff
@@ -106,7 +106,8 @@ spec:
                               "temp://lsst/dp02_dc2/hips/images/band_r",
                               "temp://lsst/dp02_dc2/hips/images/band_i",
                               "temp://lsst/dp02_dc2/hips/images/band_z",
-                              "temp://lsst/dp02_dc2/hips/images/band_y"
+                              "temp://lsst/dp02_dc2/hips/images/band_y",
+                              "ivo://CDS/P/2MASS/color"
```

Comment:
Have you confirm that `ivo` will resolve to the rubin server? I don't quite remember how to how the resolve the `ivo` in this context. I can look into i,t but if you have confirmed that it will indeed hit the rubin server then it looks good.

Reply:
OK, I checked in to it. It will resolve to the rubin service if the rubin hips list service shows that it serves `/CDS/P/2MASS/color`. 

So this line is fine but you might want to confirm this with the hips list service.